src/fec_base.h

```diff
@@ -782,6 +783,9 @@ void FecCode<T>::decode_vec_lagrange(
     if (this->fft == nullptr) {
         throw LogicError("FEC base: FFT must be initialized");
     }
+    if (this->fft_full == nullptr) {
+        throw LogicError("FEC base: FFT must be initialized");
```

Comment:
Same error message as above, but difference root cause: that can be misleading (could be "FEC base: FFT full must be initialized").